Datamigratie.Server/Features/MigrateZaak/MigrateZaakService.cs

```diff
@@ -90,6 +95,42 @@ private static void CheckIfZaakAlreadyExists()
             // TODO -> story DATA-48
         }
 
+        private async Task UploadZaakgegevensPdfAsync(DetZaak detZaak, OzZaak createdZaak, Uri informatieObjectType, CancellationToken token)
+        {
+            var pdfBytes = pdfGenerator.GenerateZaakgegevensPdf(detZaak);
+            var fileName = $"zaakgegevens_{detZaak.FunctioneleIdentificatie}.pdf";
+
+            var ozDocument = new OzDocument
+            {
+                Bestandsnaam = fileName,
+                Bronorganisatie = "999990639",
+                Formaat = "application/pdf",
+                Identificatie = $"zaakgegevens-{detZaak.FunctioneleIdentificatie}",
+                Informatieobjecttype = informatieObjectType,
+                Taal = "dut",
+                Titel = $"Zaakgegevens {detZaak.FunctioneleIdentificatie}",
+                Vertrouwelijkheidaanduiding = VertrouwelijkheidsAanduiding.openbaar,
+                Bestandsomvang = pdfBytes.Length,
+                Auteur = "Datamigratie",
+                Beschrijving = "Automatisch gegenereerd document met basisgegevens van de zaak uit het bronsysteem",
+                Creatiedatum = DateOnly.FromDateTime(DateTime.Now),
+                Status = DocumentStatus.in_bewerking,
+                Trefwoorden = [],
+                Verschijningsvorm = "verschijningsvorm",
```

Comment:
this is dummy data for a mandatory field I presume?
Can you consult Kim and see if we can set the correct values now. And for other fields she will have to create a story

Reply:
I'm not sure what you mean. Don't see a change for this specific filed. My point is that all properties should either have there final value , or Kim should be informed that we need a future story give them

Reply:
@ismailozgen you changed the wrong one. That the mapping is used for the bijlagen. the pdf function doesn't use it. it has its own mapping inside  UploadZaakgegevensPdfAsync

After this story is merged, Youri and you probably have to align some things with youri's branch